### PR TITLE
Working with Databases Documentation Updated

### PR DESCRIPTION
**Pull request by Buddhimah**

### CODE REVIEW

File: en/docs/administer/working-with-databases.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Working with Databases
 
-All WSO2 products are shipped with embedded H2 databases for storing
+WSO2 Identity Server is shipped with a embedded H2 database for storing
```
Comment (YvonneW) [inline en/docs/administer/working-with-databases.md]: an embedded

Comment (Buddhimah) [reply en/docs/administer/working-with-databases.md]: done

File: en/docs/administer/working-with-databases.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ product pack.
 Explained below are the default databases that you will find in the
 `         database        ` directory.
 
--   **Carbon database:** `          WSO2CARBON_DB.h2.db         ` is the
+-   **Carbon database:** `          WSO2CARBON_DB.mv.db         ` is the
```
Comment (ayshsandu) [inline en/docs/administer/working-with-databases.md]: Registry and user management now goes to shared db. Carbon DB is internal use only now. It's not configurable, hence no need to give prominence.

Comment (Buddhimah) [reply en/docs/administer/working-with-databases.md]: done

**Pull request merged**